tests/integrations_setup/install_agentless_integrations.py

```diff
@@ -84,23 +83,34 @@ def generate_gcp_integration_data():
         generate_aws_integration_data(),
         generate_azure_integration_data(),
     ]
-    cspm_template = generate_policy_template(cfg=cnfg.elk_config, stream_prefix="cloud_security_posture")
+    cspm_template = generate_policy_template(
+        cfg=cnfg.elk_config,
+        stream_prefix="cloud_security_posture",
+    )
     for integration_data in integrations:
-        NAME = integration_data["name"]
-        logger.info(f"Creating {NAME} integration for policy {AGENT_POLICY_ID}")
-        package_policy = generate_package_policy(
-            cspm_template,
-            integration_data,
+        INTEGRATION_NAME = integration_data["name"]
+        AGENTLESS_INPUT = {
+            "name": f"Agentless policy for {INTEGRATION_NAME}",
+            "supports_agentless": True,
+        }
+
+        logger.info(f"Starting installation of agentless-agent {INTEGRATION_NAME} integration.")
+        agent_data, package_data = load_data(
+            cfg=cnfg.elk_config,
+            agent_input=AGENTLESS_INPUT,
+            package_input=integration_data,
             stream_name="cloud_security_posture.findings",
         )
-        package_policy["force"] = True
 
-        logger.info(f"Creating {package_policy}")
+        logger.info("Create agentless-agent policy")
+        agent_policy_id = create_agent_policy(cfg=cnfg.elk_config, json_policy=agent_data)
```

Comment:
Since creating an agentless-agent policy is a new feature, we need to handle both cases: when an agentless policy already exists and when it needs to be installed (as is the case for you now). 
There are a couple of options for this:
- We could rely on an environment variable to determine if the agentless-agent policy should be installed.
- Alternatively, we could retrieve all installed agent policies from the Fleet and check if any are already agentless.

Reply:
I believe I understand what you are saying, if I am not then please correct me.

By the time this PR gets merged, Serverless production we should not be using the older preconfigured Agentless policy solution so I don't think we need check for either solution.  

When ESS production has the new Agentless API configuration we should only need to set `TEST_AGENTLESS` to true